packages/grafana-ui/src/components/PanelChrome/PanelMenu.tsx

```diff
@@ -25,10 +26,22 @@ export function PanelMenu({
   dragClassCancel,
   menuButtonClass,
   onVisibleChange,
+  onOpenMenu,
 }: PanelMenuProps) {
   const testId = title ? selectors.components.Panels.Panel.menu(title) : `panel-menu-button`;
+
+  const handleOpeningMenu = useCallback(
+    (show: boolean) => {
```

Comment:
```suggestion
  const handleVisibiliy = useCallback(
    (isVisible: boolean) => {
```
Super small suggestion. Since the callback is more about visibility that just opening the menu, I'd rename the callback to reflect that. This is totally personal preference, so feel free to ignore 😄